src/OrbitGl/Button.cpp

```diff
@@ -31,11 +31,12 @@ namespace orbit_gl {
 const Color Button::kHighlightColor(75, 75, 75, 255);
 const Color Button::kBaseColor(68, 68, 68, 255);
 const Color Button::kTextColor(255, 255, 255, 255);
+const Color Button::kSymbolsColor(191, 191, 192, 255);
 
 Button::Button(CaptureViewElement* parent, const Viewport* viewport, const TimeGraphLayout* layout)
     : CaptureViewElement(parent, viewport, layout) {
-  SetWidth(layout->GetMinButtonSize());
-  SetHeight(layout->GetMinButtonSize());
+  SetWidth(layout->GetButtonSize());
+  SetHeight(layout->GetButtonSize());
```

Comment:
Agree with 99%, thanks for pointing out. Just to mention that I decided to make Buttons children of TimeGraph, since Timeline and Buttons are split by a margin and makes more sense having them children of TimeGraph.